functions.js

```diff
@@ -160,3 +281,18 @@
  * @return {object} circle
  */
 
+function createCircle(radius){
+/*Why does this not work :'/;
+var circle;
```

Comment:
`var circle;` declares a variable `circle` with no inital value.
`circle` should be initialized with an empty object for this method to work